Documentation/index.rst

```diff
@@ -4,6 +4,14 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. versionadded::HEAD
```

Comment:
I don't think [this does what you want](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded). As I read it, this only inserts something like `New in version 0.5: The short form.` (in [this paragraph](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso)). It does not restrict the content to the specified version. The reason it doesn't show up in the preview is likely because you pass no content to the directive.

Reply:
I should have checked the documentation properly, mybad. I think the way to go will be using their beta addons.